db/seeds.rb

```diff
@@ -5,6 +5,8 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+# User.create(name: 'Test1', email: 'test1@test.com', password: 'password', gender: 'male')
```

Comment:
Don't leave comments. Auto generated comments are ok, but we prefer not to include comments in the code.